composer.json

```diff
@@ -1,6 +1,8 @@
 {
     "name": "drupal/api_proxy_pbs",
+    "type": "drupal-module",
     "require": {
-        "drupal/api_proxy": "^2.0"
+        "drupal/api_proxy": "^2.0.2",
+        "symfony/http-client": "^5.0"
```

Comment:
@lucashodge is `symfony/http-client` already part of the drupal setup? can you confirm what version already exists?